core/modules/qmeta/QStatus.h

```diff
@@ -0,0 +1,87 @@
+/*
+ * LSST Data Management System
+ * Copyright 2019 LSST Corporation.
+ *
+ * This product includes software developed by the
+ * LSST Project (http://www.lsst.org/).
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the LSST License Statement and
+ * the GNU General Public License along with this program.  If not,
+ * see <http://www.lsstcorp.org/LegalNotices/>.
+ */
+#ifndef LSST_QSERV_QMETA_QSTATUS_H
+#define LSST_QSERV_QMETA_QSTATUS_H
+
+// System headers
+#include <map>
+#include <memory>
+#include <string>
+#include <utility>
+#include <vector>
+
+// Qserv headers
+#include "qmeta/QStats.h"
+#include "qmeta/types.h"
+
+
+namespace lsst {
+namespace qserv {
+namespace qmeta {
+
+
+class QStatus {
+public:
+	typedef std::shared_ptr<QStatus> Ptr;
+    /**
+     *  Type for representing the list of tables, first item in pair is
+     *  database name, second is table name.
+     */
+    typedef std::vector<std::pair<std::string, std::string> > TableNames;
```

Comment:
Since C++11 there is no need to have spaces in template declarations. I mean this: `> >`

Reply:
Actually, no real need for that typedef.